lib/fluent/event.rb

```diff
@@ -29,7 +33,8 @@ def each(&block)
       raise NotImplementedError, "DO NOT USE THIS CLASS directly."
     end
 
-    def to_msgpack_stream
+    def to_msgpack_stream(time_int: false)
+      return to_msgpack_stream_forced_integer if time_int
```

Comment:
IMO we can remove it right now (because it's a v0.14 new feature), and it's better to be done.
But I'm not sure and this pull-request is not about that purpose.